### DISCUSSION TITLE
Impossible to overwrite already loaded project/data

### DISCUSSION THREAD

**Discussion by duboisAM** [opengisch/QField#3757] [Bugs]
Hello everyone,

There seems to be an issue with QField these days and I'm wondering if I'm the only one having it. I imported a few days back a qfield project (.qgs) and all it's related files. I made a few changes today and wanted to replace it. Whenever I go to re-import it, I get a warning message page showing up, telling me these files are already there and asking if it can overwrite them. The problem is that only the question shows up, no actual button to respond (see image below)

I rebooted the device and made sure the app is up to date but nothing seems to work.

Any suggestions? Thanks a lot!

<img width="213" alt="image" src="https://user-images.githubusercontent.com/64656876/208524156-6fe24e23-a8f7-4435-8c16-de4509f6e588.png">

**Comment by nirvn** [comment]
I believe you are hit by a bug that's been affecting a small number of samsung(?) devices. There are actually two buttons at the bottom right corner of this message box but your device is rendering the button text in white. White on white isn't good :) 

The [ overwrite ] buton would be at the right end of a bottom row, you can hit that area and it'll work.
